README.md

```diff
@@ -171,6 +186,39 @@ CONFIG:
   pooling_api: http://vmpooler.delivery.puppetlabs.net/
 ```
 
+### Custom hypervisor
+
+The following example shows one way of generating a custom hypervisor that
+includes both per-host configuration and global configuration.
+
+The term "custom" in this case signifies that it's not a built-in hypervisor
+(like `vmpooler`), which means we'll have to provide all the configuration
+ourselves as there isn't any built-in configuration for our hypervisor.
+
+```
+$ beaker-hostgenerator --hypervisor=custom --global={custom_api=http://api.custom.net} centos6-64{hypervisor=custom}
```

Comment:
They are both required.  

The `--global` is to provide the `custom_api` value in the `CONFIG` section, so you don't have to specify it on every single host.  This is akin to the `pooler_api` value we generated when specifying a `vmpooler` hypervisor.

The host-specific setting is to get the `hypervisor: custom` value on the hosts, which is necessary since hypervisors are host-specific.